spec/requests/api/submissions/index_spec.rb

```diff
@@ -31,6 +31,24 @@
       expect(body).to eq []
     end
 
+
+    fit "returns only draft submissions when state is draft" do
+      submission = Fabricate(:submission, user: user, state: "draft")
+      Fabricate(:submission, user: user, state: "approved")
+      Fabricate(:submission, user: user, state: "submitted")
+
+      get "/api/submissions",
+        params: {
+          state: "draft"
```

Comment:
**suggestion**: As implemented, the API endpoint should also support an array of state values to filter by but it would be nice to have a separate spec here to exercise that. I imagine that the GraphQL addition would also support one or more values to filter by.

```ruby
get "/api/submissions",
  params: {
    state: ["submitted", "approved"]
  },
  headers: headers
```

Reply:
I think I left a comment about this somewhere else. Would it make sense to have the state as an array here? I see we have both patterns of `state` and `states` in MP.

Reply:
I think `state` is fine. REST clients can pass a single value `state=draft` or multiple values `state[]=submitted&state[]=approved`.

For MP/GraphQL, I also think multiple values can be supported:

```graphql
{
  submissionsConnection(first: 1, state: [DRAFT])
  submissionsConnection(first: 1, state: [SUBMITTED, APPROVED])
}
```